angular.json

```diff
@@ -31,16 +34,6 @@
               "src/ng-select/lib/ng-templates.directive.ts"
             ]
           }
-        },
-        "lint": {
-          "builder": "@angular-devkit/build-angular:tslint",
```

Comment:
Should eslint not be enabled instead of just dropping tslint? Specifically builder: `@angular-eslint/builder:lint`

Reply:
This looks like why the CI is failing too since there is now no lint command.

Reply:
@pavankjadda can you address this?

Reply:
shall I add `@angular-eslint` to the project and remove tslint?

Reply:
okay. I added `angular-eslint` to the project